demo/metaprotocol-brpc/brpc-sample.yaml

```diff
@@ -28,7 +28,7 @@ spec:
     metadata:
       annotations:
         sidecar.istio.io/bootstrapOverride: aeraki-bootstrap-config
-        sidecar.istio.io/proxyImage: ghcr.io/aeraki-mesh/meta-protocol-proxy-debug:1.4.0
+        sidecar.istio.io/proxyImage: ghcr.io/aeraki-mesh/meta-protocol-proxy-debug:1.4.1
```

Comment:
To be honest, I personally feel that the  image ghcr.io/aeraki-mesh/meta-protocol-proxy:1.4.1 is more suitable here, the debug name is misleading, and the image size is much larger than it.

Reply:
Good point, let's change to then non-debug image then.